doozerlib/cli/images_streams.py

```diff
@@ -734,6 +734,7 @@ def search_issues(query):
             'project': {'key': project},
             'issuetype': {'name': 'Bug'},
             'versions': [{'name': release_version}],  # Affects Version/s
+            'customfield_12323140': [{'name': Model(runtime.gitdata.load_data(key='bug').data).jira_config.target_release[-1]}],  # Target Version
```

Comment:
Would like a comment stating that `customfield_12323140` corresponds to `Target Release` field in Jira.

Reply:
updated